jlm/llvm/backend/rvsdg2jlm/rvsdg2jlm.cpp

```diff
@@ -0,0 +1,601 @@
+/*
+ * Copyright 2017 Nico Reißmann <nico.reissmann@gmail.com>
+ * See COPYING for terms of redistribution.
+ */
+
```

Comment:
This file should not be here. It was moved in this PR #812. It should not be part of this commit.

Reply:
yes this accidentally happened during rebase conflict resolution, removing